pynio/service.py

```diff
@@ -155,3 +155,13 @@ def delete(self):
     @property
     def pid(self):
         return self._status()['pid']
+
+    def copy(self, name):
```

Comment:
Why you block.copy and service.copy? Wouldn't a user just make a new Service with some other service's config?

Service('new_service', config=service.config)